docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
+* Login uses government-issued IDIR identities
+* Users will be automatically assigned new GitHub IDs, distinct from public GitHub IDs 
+* GitHub Actions is enabled
+* Code Dependency Insights is enabled
 
-* Two-factor authentication is mandatory
-* GitHub Actions is enabled for all
-* Code Dependency Insights is enabled for all organizations linked to the GitHub Enterprise account
+## GitHub Enterprise account ownership
 
-## Request GitHub Enterprise user licences
+The B.C. government's GitHub Enterprise account is managed centrally by the Digital Office's [Developer Experience Team](mailto:developer.experience@gov.bc.ca). Ministries or sectors are provided with GitHub "organizations" within the central GitHub Enterprise account. Teams' private repositories are created within these organizations.  Management of ministry or sector organizations within GitHub Enterprise is delegated to staff within the corresponding IMB/ISB. 
 
-Contact Software Central Management at SoftwareCentral.Management@gov.bc.ca and cc Dean Picton (Dean.Picton@gov.bc.ca). Use the subject line "GitHub Enterprise Request" and send the following information:  
+Paid licenses are required to use GitHub Enterprise. Licenses must be purchased for each user that will use or administer to the B.C. government GitHub Enterprise organizations or repositories. Each ministry or sector is responsible for acquiring and paying for its own licenses. The process for acquiring licences is described below.
 
-* Private GitHub organization name and whether you need a new organization created or are linking an existing private organization  
+If you don't know whether your ministry or sector is using GitHub Enterprise or who to contact within your IMB/ISB, please [contact the Developer Experience Team](developer.experience@gov.bc.ca) for assistance.
 
-* GitHub IDs of the organization administrators
+## Acquiring GitHub Enterprise user licences
 
-* Number of user licenses needed. If you're transferring a GitHub organization to Enterprise, request as many user licenses as you have organization members.
+> Note: the process outlined below may be slightly different for some sectors or ministries, but it will be generally similar for all. For example, in some areas, the responsibility for interacting with CSAM may be done by the ministry or sector IMB/ISB staff rather than end users in a business area.
 
-## Access the GitHub Enterprise account
+Licenses for GitHub Enterprise must be purchased through OCIO Software Central Management. If you have determined that you or others in your area require licenses, the first step is to request a quote via email to  [SoftwareCentral.Management@gov.bc.ca](mailto:SoftwareCentral.Management@gov.bc.ca). Use the subject line "Request for Quotation for GitHub Enterprise"  and indicate in the body the number of users who need GitHub Enterprise licenses. For example, "Could I please get a quote for 10 GitHub Enterprises licenses (Standalone)?".
```

Comment:
1. Determine your team's needs for licenses. 
2. request a quote via email to  [SoftwareCentral.Management@gov.bc.ca](mailto:SoftwareCentral.Management@gov.bc.ca). Use the subject line "Request for Quotation for GitHub Enterprise"  and indicate in the body the number of users who need GitHub Enterprise licenses. For example, "Could I please get a quote for 10 GitHub Enterprises licenses (Standalone)?".
3. Work with your business area to determine who will have the responsibility to create an iStore order when the licenses are ready to be assigned. 
4. Once you have received a response from CSAM with a quote number and the cost, initiate the purchase & e-approvals process via iStore. 
5. Wait for CSAM to notify you by email once they have made the purchase on your behalf. 
6. Billing back to your expense authority will commence. Billing occurs monthly until you notify CSAM to cancel the user licenses.
7. Your ministry administrator will connect your subscription to and IDIR together via Azure AD to grant you access to GitHub.